templates/index.html

```diff
@@ -24,18 +24,32 @@
     </head>
 
     <body class="hack dark main container">
+        {% set lang_base_url = get_url(path="", lang=lang) %}
         {% block content %}
             {% block header %}
-                {% if config.extra.after_dark_menu %}
+                {% if config.extra.after_dark_menu or config.extra.after_dark_langs %}
```

Comment:
you don't need that with 0.16, you can just access `section.translations`. I'll release 0.16 today or tomorrow

Reply:
im not sure what you mean by this. are you suggesting: `{% if config.extra.after_dark_menu or section.translations %}`?

Reply:
I meant you can get rid of config.extra.after_dark_langs entirely